pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
```

Comment:
I don't think this setup is correct. You appear to use the `src-layout` (https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#src-layout) but have not places your content in a named subdirectory. I would guess this code installs 3 root packages named `__init__`, `dataloader`, and `run`. When it properly should install just one package named `epik_eval`.

Reply:
Fixed